app/src/main/res/layout/activity_login.xml

```diff
@@ -29,162 +29,179 @@
             app:layout_constraintTop_toBottomOf="@+id/loginImageOne" />
 
         <com.google.android.material.textfield.TextInputEditText
             android:id="@+id/loginEtUser"
-            android:layout_width="319dp"
-            android:layout_height="60dp"
-            android:layout_marginTop="5dp"
-            android:layout_marginEnd="30dp"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/size_5"
+            android:layout_marginEnd="@dimen/size_40"
+            android:layout_marginStart="@dimen/size_40"
             android:background="@drawable/style_border"
             android:backgroundTint="@color/white"
+            android:textSize="@dimen/size_text_20"
             android:inputType="textEmailAddress"
             app:layout_constraintEnd_toEndOf="parent"
-            app:layout_constraintHorizontal_bias="0.0"
             app:layout_constraintStart_toStartOf="@+id/loginTvUser"
             app:layout_constraintTop_toBottomOf="@+id/loginTvUser"
             tools:ignore="SpeakableTextPresentCheck" />
 
         <com.google.android.material.textview.MaterialTextView
-            android:id="@+id/loginTvpass"
+            android:id="@+id/loginTvPass"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:layout_marginTop="10dp"
             android:text="@string/login_tv_pass"
+            android:textSize="@dimen/size_text_20"
+            android:layout_marginStart="@dimen/size_40"
             android:textStyle="bold"
-            app:layout_constraintStart_toStartOf="@+id/loginEtUser"
+            app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toBottomOf="@+id/loginEtUser" />
 
         <com.google.android.material.textfield.TextInputEditText
-            android:id="@+id/loginEtpass"
-            android:layout_width="319dp"
-            android:layout_height="60dp"
-            android:layout_marginTop="5dp"
+            android:id="@+id/loginEtPass"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/size_5"
             android:background="@drawable/style_border"
             android:backgroundTint="@color/white"
             android:inputType="textWebPassword"
+            android:textSize="@dimen/size_text_20"
+            android:layout_marginStart="@dimen/size_40"
+            android:layout_marginEnd="@dimen/size_40"
             app:layout_constraintEnd_toEndOf="parent"
-            app:layout_constraintHorizontal_bias="0.0"
-            app:layout_constraintStart_toStartOf="@+id/loginTvpass"
-            app:layout_constraintTop_toBottomOf="@+id/loginTvpass"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/loginTvPass"
```

Comment:
when referencing existing ids avoid using the `+`, so this will also work as `"@id/loginTvPass"`